dcicutils/common.py

```diff
@@ -10,6 +10,7 @@
 
 APP_CGAP = 'cgap'
 APP_FOURFRONT = 'fourfront'
+APP_SMAHT = 'smaht'
```

Comment:
You need to update lines 25 and 35 as well. See comment on lines 31-32. (I'm unable to attach a comment there.) It might be good to add a comment here that says
```
# If this set is updated, the values of ORCHESTRATED_APPS and OrchestratedApp need to be updated.
```